k9mail/src/main/java/com/fsck/k9/helper/FileHelper.java

```diff
@@ -147,6 +141,11 @@ public static boolean move(final File from, final File to) {
 
         try {
             copyFile(from, to);
+
+            boolean deleteFromOk = from.delete();
```

Comment:
Hm. I like having methods which are called for their side effects as stand-alone as possible, rather than directly use their return value in an expression. On the other hand there is value in avoiding nontrivial expressions in the if block... hmh. :\